uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -147,11 +147,15 @@ var Promise = require('es6-promise').Promise;
         return _.assign(scenario, { delay: scenario.delay || 6000 });
       })
       .tap(function (scenarios) {
-        var previousCredential = null;
+        var DEFAULT_CREDENTIAL = 'admin';
+        var previousCredential;
 
-        scenarios.forEach(function (scenario) {
-          if (previousCredential !== scenario.credential || previousCredential === null) {
+        scenarios.forEach(function (scenario, index) {
+          scenario.credential = scenario.credential || DEFAULT_CREDENTIAL;
+
+          if (index === 0 || previousCredential !== scenario.credential) {
             scenario.onBeforeScript = 'login';
+            scenario.isLogOutNecessary = true;
```

Comment:
also, you need to set it to `false` if it's not `true`

Reply:
As discussed, not adding it.